ios-theater/View/HotMoviesFooter.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  HotMoviesFooter.swift
+//  ios-theater
+//
+//  Created by Linxiao Wei on 2019/12/30.
+//  Copyright © 2019 Linxiao Wei. All rights reserved.
+//
+
+import UIKit
+
+class HotMoviesFooter: UITableViewHeaderFooterView {
+  @IBOutlet weak var indicater: UIActivityIndicatorView!
+  @IBOutlet weak var endLabel: UILabel!
+  
+  func configure() {
```

Comment:
This name doesn't make sense, and why not hide these views in `xib` directly?

Reply:
Okay, I will rename them and hide these views in `xib`.